### ISSUE TITLE
Never ending hourglass on raster identify

### ISSUE THREAD

**Issue by gioman** [3liz/lizmap-web-client#1719]
Having a Lizmap/QGIS project with a identifiable single band raster, when the user click on the map to get the raster value this is returned, but the mouse pointer become a unstoppable hourglass (when hovering over the map). Beside this annoyance LMWC is still functional.

LMWC 3.3.8
QGIS Server 3.10 on Ubuntu Server 20.04
Firefox 78.0.1

**Comment by nboisteault** [comment]
Is there any message in console log ?

**Comment by gioman** [comment]
@nboisteault 

in the LMWC log

```

2020-07-17 12:09:16	85.242.45.64	warning	2020-07-17 12:09:16	[2]	SQLite3::query(): Unable to prepare statement: 1, no such table: lizmap_search	/var/www/html/soil_tests/lib/jelix/plugins/db/sqlite3/sqlite3.dbconnection.php	112

2020-07-17 12:09:16	85.242.45.64	warning	2020-07-17 12:09:16	[2]	file_get_contents(http://127.0.0.1/cgi-bin/qgis_mapserv.fcgi?service=WMS&amp;request=GetCapabilitiesAtlas&amp;map=%2Fvar%2Fwww%2Fhtml%2Fsoil%5Ftests%2Flizmap%2Finstall%2Fsoiltests%2Fsoil%5Fmapping%5Fwebremote%5Fwu%2Eqgs): failed to open stream: HTTP request failed! HTTP/1.1 501 Not Implemented
	/var/www/html/soil_tests/lizmap/modules/lizmap/classes/lizmapProxy.class.php	240
```

In FF web console I don't see any request when identyfing a raster.


I see the same behaviour on Chrome, I that matters.

**Comment by aeduard** [comment]
I have the same errors without having any raster layer on lizmap map. It gets written in the log file when loading a map.

Configuration very similar to gioman's setup:

LMWC 3.3.8
QGIS Server 3.10 on Ubuntu Server 20.04
Firefox 78.0.2

**Comment by nboisteault** [comment]
@gioman @aeduard Could you provide a minimal project which reproduces the issue ?

**Comment by gioman** [comment]
@nboisteault https://training.naturalgis.pt/soil_tests/lizmap/www/index.php/view/map/?repository=soilstests&project=soil_mapping_webremote_wu which can be used as example also for https://github.com/3liz/lizmap-web-client/issues/1720

**Comment by nboisteault** [comment]
@gioman When I click nothing happens (no request in network tab) neither in Chrome nor Firefox.

**Comment by gioman** [comment]
> @gioman When I click nothing happens (no request in network tab) neither in Chrome nor Firefox.

@nboisteault see attached screencast:

![Peek 2020-07-23 15-01](https://user-images.githubusercontent.com/1951107/88295594-880f9d80-ccf5-11ea-9588-1e01c7b65946.gif)

**Comment by gioman** [comment]
@nboisteault sorry the screencast didn't recorded the mouse pointer. Let me try again.

**Comment by gioman** [comment]
@nboisteault sorry, no Linux screen recorder captured the mouse pointer while it becomes a hourglass. I had to take the video with my phone:

https://drive.google.com/file/d/1DzcVsJU0ltXYWAt_ipUiEHDiv5B6Dexu/view?usp=sharing

**Comment by nboisteault** [comment]
@gioman ok I understand, I'll fix it.

**Comment by nboisteault** [comment]
@gioman It should be ok with last commit above in release_3_3 branch.

**Comment by gioman** [comment]
> @gioman It should be ok with last commit above in release_3_3 branch.

@nboisteault tested, works fine! thanks!

**Comment by aeduard** [comment]
I'm not sure if I did what I should had to...I've updated to lizmap version 3.3.9. Update made by downloading the zip file, and made the procedure according to documentation ([link](https://docs.lizmap.com/next/en/install/upgrade.html)). Upgrade looks successful in console, but error remains. Also administration page still writes version 3.3.8.
Well I didn't had the hourglass issue (not tested, because I don't have raster in my project).

2020-07-23 22:56:14	_removed ip_	warning	2020-07-23 22:56:14	[2]	SQLite3::query(): Unable to prepare statement: 1, no such table: lizmap_search	/var/www/lizmap-web-client-3.3.9/lib/jelix/plugins/db/sqlite3/sqlite3.dbconnection.php	112

2020-07-23 22:56:14	_removed ip_	warning	2020-07-23 22:56:14	[2]	file_get_contents(http://127.0.0.1/cgi-bin/qgis_mapserv.fcgi?service=WMS&amp;request=GetCapabilitiesAtlas&amp;map=%2Fhome%2F_(removed)_%2F_(removed)_%2Eqgs): failed to open stream: HTTP request failed! HTTP/1.1 501 Not Implemented
	/var/www/lizmap-web-client-3.3.9/lizmap/modules/lizmap/classes/lizmapProxy.class.php	240

**Comment by nboisteault** [comment]
@aeduard Fix is not released yet. But you can test it in release_3_3 branch with git or patch your code by looking at the diff (it is not a lot of code).

**Comment by aeduard** [comment]
Thank you! It's not a great issue for me, I'm waiting for the fix.

**Comment by josemvm** [comment]
hi @nboisteault i still have the same problem, even after this commityour fix https://github.com/3liz/lizmap-web-client/commit/1f8f7eacd78f7b7b51589256a15cf1d0937836fe

i have no errors but the hourglass never endings...

**Comment by nboisteault** [comment]
@josemvm Any public link ? No errors even in browser's console?

**Comment by josemvm** [comment]
@nboisteault please check your mailbox

**Comment by nboisteault** [comment]
@josemvm Please pull release_3_3, it should be ok now.

**Comment by josemvm** [comment]
@nboisteault 

now i have this error in another site:

![imagem](https://user-images.githubusercontent.com/10053874/88382987-83a0be80-cda1-11ea-8961-9fad46de434a.png)

**Comment by nboisteault** [comment]
Are you on release_3_3 on this second site ?

**Comment by josemvm** [comment]
yes, i have updated all my sites...

**Comment by nboisteault** [comment]
Can you share a link ?

**Comment by josemvm** [comment]
yes, please check again your mailbox

**Comment by josemvm** [comment]
thanks @nboisteault now everything works fine!!! 👍 
many thanks
